src/qlik-date-picker.js

```diff
@@ -71,6 +71,17 @@ define(["qlik", "jquery", "./lib/moment.min", "./calendar-settings", "css!./lib/
             html += '</div>';
             return html;
         }
+        function getPosition( element ) {
+            if (element.offset().left < 600) {
+                return "right";
+            }
+            else if (element.offset().right < 600) {
```

Comment:
the "else if" and "else" both end up with the same result, so then a simple "else" should be enough.